### DISCUSSION TITLE
本地部署文章/页面发布后不能在对应的控件页面不能看到最新的

### DISCUSSION THREAD

**Discussion by inkchills** [Qexo/Qexo#340] [Ideas]
只有点击清除缓存才能看得到，建议调用部署之后调用一下清除缓存功能，我自己去测试了，这个是可以实现的

**Comment by am-abudu** [comment]
已经有更好的实现方法 请参见[文档](https://www.oplog.cn/qexo/start/questions.html#%E4%B8%BA%E4%BB%80%E4%B9%88%E6%88%91%E6%96%B0%E5%BB%BA%E7%9A%84%E6%96%87%E7%AB%A0%E6%B2%A1%E6%9C%89%E6%98%BE%E7%A4%BA-qexo-%E5%A6%82%E4%BD%95%E4%BF%9D%E8%AF%81%E6%96%87%E4%BB%B6%E6%9C%80%E6%96%B0)

**Reply by inkchills** [reply]
本地部署点击一键设置WebHook提示服务商不支持

**Reply by am-abudu** [reply]
我会继续优化这里的逻辑

**Comment by am-abudu** [comment]
我稍微修改了一下，你可以试试Dev分支

**Reply by inkchills** [reply]
dev分支，我刚刚试了还是提示服务商不支持
